Jenkinsfile

```diff
@@ -161,7 +166,7 @@ def doBuildDotNetOsx() {
               sh build.sh config $dir/install
               sh build.sh build-dotnet-cocoa
```

Comment:
setting an environment variable and prepending a command with it should is the same thing:

```
$ env | grep TEST
$» TEST=1 env | grep TEST
TEST=1
```

So we need to figure out why the environment variable is not set correctly in the current setup. I Imagine it's some escaping issue.